subsys/mpsl/fem/Kconfig

```diff
@@ -17,9 +17,7 @@ config MPSL_FEM_GENERIC_TWO_CTRL_PINS_SUPPORT
 
 config MPSL_FEM
 	bool "Radio front-end module (FEM) support"
-# MPSL FEM is not supported on 53 yet
-	depends on !(SOC_NRF5340_CPUAPP || SOC_NRF5340_CPUNET) && \
-		   (MPSL_FEM_NRF21540_GPIO_SUPPORT || MPSL_FEM_GENERIC_TWO_CTRL_PINS_SUPPORT)
+	depends on (MPSL_FEM_NRF21540_GPIO_SUPPORT || MPSL_FEM_GENERIC_TWO_CTRL_PINS_SUPPORT)
```

Comment:
Please keep MPSL_FEM disabled for MPSL_FEM_GENERIC_TWO_CTRL_PINS_SUPPORT as before until it is supported by nRF5340. Since this change, it brings a compiling error when building "generic-fem-two-ctrl-pins" on nRF5340.

Reply:
I've added the dependency you requested. Please check if the patch meets your requirements now

Reply:
verified.